datajoint/fetch.py

```diff
@@ -199,15 +153,11 @@ def __getitem__(self, item):
         :return: tuple with an entry for each element of item
 
         Examples:
-
         >>> a, b = relation['a', 'b']
         >>> a, b, key = relation['a', 'b', datajoint.key]
-        >>> results = relation['a':'z']    # return attributes a-z as a tuple
```

Comment:
yes, slicing and number indexing of attributes is an invitation to write bad code, relying on the order and number of attributes in a relation rather their names.  This feature adds the testing and documentation load while its functionality could be straightforwardly accomplished by other, more explicit means.